ansible/roles/integration_tests/tasks/tools.yml

```diff
@@ -10,7 +10,7 @@
 
     - name: Download and extract the tkn binary
       ansible.builtin.unarchive:
-        src: https://mirror.openshift.com/pub/openshift-v4/clients/pipelines/1.10.0/tkn-linux-amd64.tar.gz
+        src: https://mirror.openshift.com/pub/openshift-v4/clients/pipelines/1.11.2/arm64/linux/tkn-linux-arm64.tar.gz
```

Comment:
I think you pasted the wrong URL here: this is the arm64 binary tarball. We need the amd64 version.

Reply:
Good catch! I fixed the URL